packages/metrics/tests/e2e/standardFunctions.test.MyFunction.ts

```diff
@@ -1,7 +1,7 @@
 import { Metrics, MetricUnits } from '../../src';
 import { Context } from 'aws-lambda';
 
-const namespace = process.env.EXPECTED_NAMESPACE ?? 'CDKExample';
+const namespace = process.env.EXPECTED_NAMESPACE ?? 'LambdaPowertoolsTypeScript-CDKExample';
```

Comment:
Same as other file, I'm open to discuss the change but in a separate PR, so to keep this one focused on examples.